### PR TITLE
fix lang fa

### PR DESCRIPTION
**Pull request by KarimQaderi**
fix lang fa

### CODE REVIEW

**Comment by KarimQaderi** [comment]
Hi, I tested my system, there was no problem

File: packages/Webkul/Ui/src/DataGrid/DataGrid.php [hunk 1]
```diff
@@ -283,6 +284,8 @@ public function addAction($action, $specialPermission = false)
         $this->checkPermissions($action, $specialPermission, function ($action, $eventName) {
             $this->fireEvent('action.before.' . $eventName);
 
+            $action['key'] = Str::slug($action['title'], '_');
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/DataGrid/DataGrid.php:287]: Just keep it simple and try to move this one in the transform action method.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by devansh-webkul** [comment]
Hi @KarimQaderi,

I just reviewed the full PR, but still, I want you to do two separate PR for that first one for lang and the second one for Datagrid.

File: packages/Webkul/Ui/src/DataGrid/DataGrid.php [hunk 2]
```diff
@@ -5,6 +5,7 @@
 use Illuminate\Support\Facades\Event;
 use Webkul\Ui\DataGrid\Traits\ProvideBouncer;
 use Webkul\Ui\DataGrid\Traits\ProvideCollection;
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/DataGrid/DataGrid.php:7]: Sort this import also in your new PR.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Ui/src/DataGrid/DataGrid.php [hunk 3]
```diff
@@ -5,6 +5,7 @@
...
+use Illuminate\Support\Str;
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/DataGrid/DataGrid.php:8]: Sort this import also in your new PR.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Pull request closed without merging**